contracts/BridgeDepositBox.sol

```diff
@@ -0,0 +1,293 @@
+// SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
where did this file come from? doesnt seem needed for oo

also the few .sol files included, can they not be imported?